source/robot_model/src/Model.cpp

```diff
@@ -41,6 +41,17 @@ Model::Model(const Model& model) :
 
 Model::~Model() {}
 
+void Model::init_model() {
+  pinocchio::urdf::buildModel(this->get_urdf_path(), this->robot_model_);
+  this->robot_data_ = pinocchio::Data(this->robot_model_);
+  // get the frame names
+  std::vector<std::string> frames;
+  for (auto& f : this->robot_model_.frames) {
+    frames.push_back(f.name);
+  }
+  this->frame_names_ = std::vector<std::string>(frames.begin() + 2, frames.end());
```

Comment:
maybe add a brief comment to say that you are intentionally skipping `universe` and `root_joint`? (since in get_joint_frames you justify omitting universe)

Reply:
indeed thanks